variables.tf

```diff
@@ -0,0 +1,27 @@
+
+variable "location" {
+ description = "The Azure location where all resources in thisdeployment should be created"
+ default = "uksouth"
+}
+variable "SECRET_KEY" {
+    default = "secret-key"
```

Comment:
This should also be sensitive. It's used to encrypt the session cookie, so if it leaked anyone could pretend to be logged into your to do app.

Reply:
Value moved to `terraform.tfvars` which will be ignored.

Reply:
(In the upcoming Pull request)